pricing.html

```diff
@@ -135,110 +135,116 @@ <h3 class="separator dark-text-title mainHead" style="float:left; width:100%; ma
+                <div class="mus-hours">Per month flat fee includes up to <b>667</b> studio hours</div>
                 <div class="addPrice">+$0.45</div>
-                <div class="mus-hours-full1">Per additional musician/hour above <b>667</b></div>
+                <div class="mus-hours-full1">Per additional studio hour above <b>667</b></div>
                 <div class="mus-hours-full2">Billed monthly, in 1-minute increments</div>
             </div>
             <div class="sub" style="background-color: #1D4A6F;">
                 <div class="sub_title">Amphitheater</div>
                 <div class="subPrice" id="c7Txt" style="font-weight: 600;">$500.00</div>
-                <div class="mus-hours">Per month flat fee for up to <b>1,500</b> musician/hours</div>
+                <div class="mus-hours">Per month flat fee includes up to <b>1,500</b> studio hours</div>
                 <div class="addPrice">+$0.40</div>
-                <div class="mus-hours-full1">Per additional musician/hour above <b>1,500</b></div>
+                <div class="mus-hours-full1">Per additional studio hour above <b>1,500</b></div>
                 <div class="mus-hours-full2">Billed monthly, in 1-minute increments</div>
             </div>
             <div class="sub" style="background-color: #1D4A6F;">
                 <div class="sub_title">Stadium</div>
                 <div class="subPrice" id="c8Txt" style="font-weight: 600;">$1000.00</div>
-                <div class="mus-hours">Per month flat fee for up to <b>3,429</b> musician/hours</div>
+                <div class="mus-hours">Per month flat fee includes up to <b>3,429</b> studio hours</div>
                 <div class="addPrice">+$0.35</div>
-                <div class="mus-hours-full1">Per additional musician/hour above <b>3,429</b></div>
+                <div class="mus-hours-full1">Per additional studio hour above <b>3,429</b></div>
                 <div class="mus-hours-full2">Billed monthly, in 1-minute increments</div>
             </div>
         </div>
     </section>
 
     <!-- <section id="priceplansubtxt" class="priceplansubtxt main" style="margin-bottom:100px;">
         <div class="left-side-content title-line">
-           <p class="italicTxt">* Note: a minimum $1/month subscription is required to host managed audio servers</p>
-           <p class="italicTxt">* Note: 1 musician/minute is consumed for every unique device or client connected to a managed audio server during a 1-minute interval</p>
-           <p class="italicTxt">* Note: each room size has a minimum number of musicians/minute (see below); this is the minimum amount consumed while running a server</p>
-           <p class="italicTxt">* Note: no subscription is required to host unmanaged audio servers, or connect to audio servers hosted by someone else</p>
+           <p class="italicTxt">* Note: a minimum $1/month subscription is required to host managed virtual studios</p>
```

Comment:
This isn't true any more, right? There's a $0 plan that you could theoretically run for 10 minutes with a friend and not be charged?

Reply:
this part of the code is commented out